app/Http/Controllers/Admin/KitchenController.php

```diff
@@ -75,7 +75,7 @@ public function destroy(Kitchen $kitchen) {
 	}
 
 	public function fields(){
-        $fields  = Kitchen::fields()->sortBy('order');
+        $fields  = json_encode(Kitchen::fields());
```

Comment:
no need to json_encode this - the method returns a collection, which prints wihtout encoding